modulefiles/modulefile.hafs.hera

```diff
@@ -0,0 +1,71 @@
+#%Module#####################################################
+## Module file for hafs
```

Comment:
Since the intent is to add the standalone Cmake build system, it would be better to change hafs/HAFS into tracker or something similar.

Also, you may want to change the modulefile name into modulefile.hera or modulefile.tracker.hera.

Reply:
@BinLiu-NOAA  Thanks. 

Should we use component level modulefile(hafs/sorc/hafs_vortextracker.fd/modulefiles) or top-level modulefile(hafs/modulefiles) during running binaries for the tracker?

Reply:
@BijuThomas-NOAA, In the HAFS application/workflow level, if the application level modulefile has all the modules needed to run the tracker, then we do not need to load the tracker level module file again in the workflow. But, this can be changed/updated in the workflow level in the future if needed.